Dockerfile

```diff
@@ -12,19 +12,7 @@ LABEL org.opencontainers.image.ref.name="jnovack/autossh" \
       org.opencontainers.image.revision=$COMMIT \
       org.opencontainers.image.url="https://hub.docker.com/r/jnovack/autossh/"
 
-ENTRYPOINT ["/entrypoint.sh"]
-ADD /entrypoint.sh /entrypoint.sh
-RUN chmod 755 /entrypoint.sh
+RUN apk --no-cache add autossh openssh-client
+COPY /entrypoint.sh /init
 
-ENV \
-    TERM=xterm \
```

Comment:
I've had issues in the past removing TERM=xterm with `autossh`.  If that's no longer a problem in newer builds of autossh and ssh, then I am ok with keeping this removed.

Reply:
I haven't had any issues with this. Must have been a bug that was resolved at some point, I can't imagine why a non-interactive process would need `$TERM` set